### PR TITLE
Display user in terminal title

### PR DESCRIPTION
**Pull request by kephas**

### CODE REVIEW

**Comment by kephas** [comment]
I didn't see how, but avoiding the code duplication would obviously be nice here.

File: fish_title.fish [hunk 1]
```diff
@@ -3,7 +3,19 @@
 #     set -g theme_title_display_path no
 #     set -g theme_title_use_abbreviated_path no
 
+function __bobthefish_user -S -d 'Display actual user if different from $default_user'
+  if [ "$theme_display_user" = 'yes' ]
```
Comment (bobthecow) [inline fish_title.fish]: let's make this `$theme_title_display_user` so they can be controlled independently.

File: fish_title.fish [hunk 2]
```diff
@@ -3,7 +3,19 @@
 #     set -g theme_title_display_path no
 #     set -g theme_title_use_abbreviated_path no
 
+function __bobthefish_user -S -d 'Display actual user if different from $default_user'
```
Comment (bobthecow) [inline fish_title.fish]: … and then call this `__bobthefish_title_user` :)

**Comment by bobthecow** [comment]
actually, don't worry about it. i'll merge then make that change.

**Pull request merged**